install.sh

```diff
@@ -118,6 +120,16 @@ case "${ARCH}" in
   ;;
 esac
 
+libc_version=$(/lib"$LIB_SUFFIX"/libc.so.6 | head -n 1 | sed -E 's/.*(stable release version.*) (.*)./\2/')
+case "${libc_version}" in
+"2.35")
+# Prepend the correct packages for M113 onwards systems.
+BOOTSTRAP_PACKAGES="glibc_lib235 zlibpkg gmp ${BOOTSTRAP_PACKAGES}"
```

Comment:
Simplify to `[[ "${libc_version}" == "2.35" ]] && BOOTSTRAP_PACKAGES="glibc_lib235 zlibpkg gmp ${BOOTSTRAP_PACKAGES}"`?

Reply:
We will need a new case statement though when we have a newer version of glibc...
Is it better then to add a new `[[ "${libc_version}" == "2.36" ]] && BOOTSTRAP_PACKAGES="glibc_lib236 zlibpkg gmp ${BOOTSTRAP_PACKAGES}"` line or then switch to a case statement?